apis/v1/jaeger_webhook.go

```diff
@@ -48,6 +48,7 @@ func (j *Jaeger) objsWithOptions() []*Options {
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (j *Jaeger) Default() {
 	jaegerlog.Info("default", "name", j.Name)
+	jaegerlog.Info("WARNING jaeger-agent is deprecated and will removed in v1.55.0. See https://github.com/jaegertracing/jaeger/issues/4739", "component", "agent")
```

Comment:
Just for the record, I added it here and not in the webhook for the sidecar injection since its the agent can also be deployed as a daemonset.

Reply:
Please update the linked issue in the root repository once the changes are done in the operator